languages/nias/apk/release/output.json

```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":208,"versionName":"4.0.208","enabled":true,"outputFile":"ASKLangPack-nias-208.apk","fullName":"release","baseName":"release"},"path":"ASKLangPack-nias-208.apk","properties":{}}]
```

Comment:
this file should be removed

Reply:
Oh sorry, I don't know how it was being staged as I put the directory release/ in .gitignore.

I have removed the file in case. Would you be kind to help tell me whether I need to close the pull request first and then submit a new one? Sorry I am still learning...